src/components/pages/bc-send-payment.ts

```diff
@@ -39,6 +40,38 @@ export class SendPayment extends withTwind()(BitcoinConnectElement) {
   })
   invoice?: string;
 
+  @state()
+  _fiatValue?: string;
+
+  private async fetchFiatValue() {
```

Comment:
This code here and in bc-start is almost duplicated (some minor changes in naming), and if we want to show fiat somewhere else it would be duplicated again. How can we extract it?

Ideally we would just do `this._fiatValue = getFiatValue(decodedInvoice.satoshi);`

Reply:
Actually, I think we should make a separate component for fiat balance display (then we also address the duplication issue.)

Reply:
Something like:

```
<bci-fiat-balance value={decodedInvoice.satoshi}></bci-fiat-balance>;
```

(bci is the prefix we use for internal web components)